Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -1523,6 +1738,146 @@ def get_context_table_csv(client: Client, args: Dict[str, str]) -> Tuple[Any, Di
     return f'Successfully downloaded Context Table CSV file {context_table_name}.', {}, None
 
 
+def get_notable_assets(client: Client, args: Dict) -> Tuple[Any, Dict[str, Any], Optional[Any]]:
+    """  Updates records of a context table.
+
+    Args:
+        client: Client
+        args: Dict
+
+    """
+    limit: int = args.get('limit', 10)
+    time_period: str = args.get('time_period', '')
+    time_ = time_period.split(' ')
+    if not len(time_) == 2:
+        raise Exception('Got invalid time period. Enter the time period number and unit.')
+    num: str = time_[0]
+    unit: str = time_[1]
+    api_unit = unit[0]
+    if api_unit == 'm':
+        api_unit = api_unit.upper()
+
+    if api_unit not in {'d', 'y', 'M', 'h'}:
+        raise Exception('The time unit is incorrect - can be hours, days, months, years.')
+
+    notable_assets_raw_data = client.get_notable_assets_request(api_unit, num, limit)
+    notable_assets = notable_assets_raw_data.get('assets')
+
+    if not notable_assets:
+        return 'No users were found in this period of time.', {}, {}
+
+    contents: list = []
+    for asset in notable_assets:
+        asset_ = asset.get('asset', {})
+        highest_risk_sequence = asset.get('highestRiskSequence', {})
+        latest_asset_comment = asset.get('latestAssetComment', {})
+        contents = contents_append_notable_assets_info(contents, asset, asset_, highest_risk_sequence,
+                                                       latest_asset_comment)
+
+    entry_context = {'Exabeam.NotableAsset(val.ipAddress && val.ipAddress === obj.ipAddress)': contents}
```

Comment:
Assets have either an IP address or a hostname value
```suggestion
    entry_context = {'Exabeam.NotableAsset((val.ipAddress && val.ipAddress === obj.ipAddress) || (val.hostName && val.hostName === obj.hostName))': contents}
```

Reply:
done